testsuite/MDAnalysisTests/analysis/test_base.py

```diff
@@ -162,6 +162,25 @@ def test_AnalysisFromFunction(u, start, stop, step, nframes):
         assert_equal(results, ana.results)
 
 
+def test_AnalysisFromFunction_args_order(u):
```

Comment:
As a general rule, one test function for one assertion _especially_ if you are testing different things such as different arguments.